hs628_tools/Outliers.py

```diff
@@ -0,0 +1,76 @@
+import pylab
+import numpy as np
+
+
+class Outliers:
+    """
+    Concept:
+    Returns list of data(list array) which fall outside of two standard deviations under the normal distribution curve.
+
+    Parameters:
+    mean, standard deviation(std)
+
+    Returns:
+    outliers
+
+    *Note:
+    Assumed the data set('data.csv') is loaded
+    """
+
+    def __init__(self, mean=0, std=1, data=None):  # default values input for mean and std
+        """
+        Actions:
+        (0) Define parameters
+        (1) Load data
+        (2) Clean data by removing none numeric values("None", "Norm", and etc.)
+        (3) Convert values from string to float
+
+        Unit tests to confirm data cleaning:
+          >>> "6".isnumeric()
+          True
+          >>> "None".isnumeric()
+          False
+          >>> "Norm".isnumeric()
+          False
+        """
+        self.mean = mean
+        self.std = std
+        self.data = data
+
+        if data is None:
+            for i in range(len(self.data)):  # remove None in data.csv
+                self.data[i] = [elem for elem in self.data[i] if elem.isnumeric()]
+            self.data = np.array(self.data, dtype=np.float)  # convert to float
+            print(self.data)
+    ##
+    def cal_outliers(my_arr, out_lim):
+        """
+        Concept:
+        Function for calculating values out of scope of two standard deviation in data and returning those values as
+        outliers
+
+        Actions:
+        Get quartile values for outlier limits: out of two standard deviation
```

Comment:
I'm not seeing how this really corresponds to quartile values.

Reply:
This is erased.